### PR TITLE
remove isPro check

### LINKED ISSUE #8963: [CAL-1718] Code Cleanup: All accounts are PRO now, so remove isProOnly flag from App Store Apps
Screenshot 2023-05-18 at 11 56 35 AM

<sub>[CAL-1718](https://linear.app/calcom/issue/CAL-1718/code-cleanup-all-accounts-are-pro-now-so-remove-isproonly-flag-from)</sub>

### PR DESCRIPTION
**Pull request by rkreddy99**
## What does this PR do?

Fixes #8963 

Remove isProOnly props from the code as every account is pro now.

Loom Video: https://www.loom.com/share/0aefcffa2a1e440c9f36a5bef719f7f7
This video shows that after removal of the isPro props, the user can still add apps 

**Environment**: Production

## Type of change

- Chore (refactoring code, technical debt, workflow improvements)

## How should this be tested?

- [ ] User should be able to add apps from app store

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

File: yarn.lock
Comment (joeauyeung) [inline yarn.lock:1]: This should be changed. Can you checkout the `yarn.lock` file from `main` to revert these changes?

**Review by joeauyeung — DISMISSED** [review]
Tested with Routing Forms app
- Installed from App Store page
- Installed from Routing Forms app page

Everything works but left a comment about the `yarn.lock` file

File: yarn.lock
Comment (rkreddy99) [reply yarn.lock:1]: done, checkedout lock file from main

File: apps/web/public/static/locales/ar/common.json [hunk 1]
```diff
@@ -1572,7 +1572,7 @@
   "sender_name": "اسم المرسل",
   "already_invited": "تمت دعوة هذا الحاضر بالفعل",
   "no_recordings_found": "لم يتم العثور على تسجيلات",
-  "new_workflow_subtitle": "سير عمل جديد من أجل...",
+  "new_workflow_subtitle": "سير عمل جديد لـ...",
```
Comment (hariombalhara) [inline apps/web/public/static/locales/ar/common.json:1575]: Why is this change here?

File: packages/app-store/ping/api/_getAdd.ts [hunk 2]
```diff
@@ -20,7 +20,7 @@ export async function getHandler(req: NextApiRequest, res: NextApiResponse) {
     key: {},
   });
 
-  return { url: getInstalledAppPath({ variant: "other", slug: "ping" }) };
+  return { url: getInstalledAppPath({ variant: appConfig.variant, slug: "ping" }) };
```
Comment (hariombalhara) [inline packages/app-store/ping/api/_getAdd.ts:23]: Same here. I think this belongs to another branch.

**Review by hariombalhara — CHANGES_REQUESTED** [review]
Looks good except for some unrelated changes in the PR.

**Review by rkreddy99 — COMMENTED** [review]
remove unnecessary changes

**Comment by hariombalhara** [comment]
@rkreddy99 Can you please resolve the conflicts once and then I would merge.

**Review by hariombalhara — APPROVED** [review]
LGTM now !! Resolved the conflicts as they were from my PR.

**Review by sean-brydon — APPROVED** [review]
LGTM on a UI side

**Review by sean-brydon — APPROVED** [review]
LGTM on a UI side

**Pull request merged**